
---- EXPECTED OUTPUT
Call show()

Fatal error: Uncaught Error: Call to private method pass::show() from context 'fail' in %s:%d
Stack trace:
#0 %s(%d): fail::not_ok()
#1 {main}
  thrown in %s on line %d
---- ACTUAL OUTPUT
Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
Call show()

Fatal error: Uncaught Error: Call to private method pass::show() from context 'fail' in /root/php-src/tests/classes/private_003.php:19
Stack trace:
#0 /root/php-src/tests/classes/private_003.php(24): fail::not_ok()
#1 {main}
  thrown in /root/php-src/tests/classes/private_003.php on line 19
---- FAILED
